<launch>

   <!-- urdf specifies target chain for retargeting -->  
  <arg name="urdf_filename" default="$(find bandit_urdf)/urdf/bandit_defs.urdf" />
  <arg name="world_frame_name" default="/world" />
  
  <!-- <arg name="chain_defs" default="$(find kinematic_retargeting)/params/bandit_right_arm.yaml" /> -->
  <arg name="chain_defs" default="$(find kinematic_retargeting)/params/bandit_humanoid.yaml" />
  <!-- <arg name="chain_defs" default="$(find kinematic_retargeting)/params/pr2_right_arm.yaml" /> -->

  <arg name="pkg" value="kinematic_retargeting" />
  <arg name="type" default="humanoid_retargeter" />
  <arg name="name" default="$(arg type)" />
  <arg name="rate" default="5" />
  <arg name="args" value="_loop_rate:=$(arg rate) _urdf_uri:=$(arg urdf_filename) 
			  _chain_defs:=$(arg chain_defs) 
			  _world_frame_name:=$(arg world_frame_name)"/>

  <node
      pkg="$(arg pkg)"
      type="$(arg type)_node"
      name="$(arg name)"
      args="$(arg args)"
      output="screen" />
</launch>
